.golangci-required.yml

```diff
@@ -0,0 +1,33 @@
+---
+# The most valuable linters; they are required to pass for PR to be merged.
+
+run:
+  skip-dirs:
+    - bogus
+
+  modules-download-mode: readonly
+
+linters-settings:
+  govet:
+    check-shadowing: true
+  goimports:
+    local-prefixes: gopkg.in/reform.v1
+
+linters:
+  disable-all: true
+  enable:
+    - deadcode
+    - depguard
+    - errcheck
+    - goimports
+    - golint
+    - gosec
```

Comment:
Ok, in the main `.golangci.yml` you exclude some checks for gosec. Here you use the default config for it. Why?